src/layout/section/illustratedSection/index.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components'
+import { responsiveBreakpoints, space } from '_utils/branding'
+
+import IllustratedSection from './illustratedSection'
+
+const StyledIllustratedSection = styled(IllustratedSection)`
+  & .section-content {
+    display: flex;
+    flex-direction: column;
+  }
+
+  & .kirk-illustratedSection-illustration {
+    margin-left: -${space.xl};
+    margin-right: -${space.xl};
+    text-align: center;
+  }
+
+  & .kirk-illustratedSection-illustration img {
+    max-width: 100%;
+  }
+
+  @media (${responsiveBreakpoints.isMediaLarge}) {
+    & .section-content {
+      flex-direction: row;
+    }
+
+    & .kirk-illustratedSection-illustration {
+      margin-left: 0;
+      margin-right: 0;
+      width: 33.33%;
+    }
+
+    & .kirk-illustratedSection-content {
+      margin-left: ${space.xl};
+      width: 66.66%;
```

Comment:
Do we want a max-width here ? Otherwise on a 30inch monitor, the itemchoices will look quite weird (text of the right and the chevron will be 50 cm away).
The same for the illustration, we probably don't want to grow indefinitely ?
This can be done iteratively in a future PR if needed.

Reply:
Actually you have a wrapping BaseSection so that should work, diimiss my comment :pray: